.envrc

```diff
@@ -0,0 +1 @@
+use nix
```

Comment:
I see some `direnv reload` instructions below. If you add the relevant files here with `watch_file` then it will automatically reload as necessary.

```
watch_file nix/gemset.nix
```

etc

Reply:
Will do, thanks!